ui/components/app/snaps/snap-remove-warning/snap-remove-warning.js

```diff
@@ -1,51 +1,67 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useI18nContext } from '../../../../hooks/useI18nContext';
-import Typography from '../../../ui/typography/typography';
-import { TypographyVariant } from '../../../../helpers/constants/design-system';
-import Box from '../../../ui/box/box';
-import Popover from '../../../ui/popover';
-import Button from '../../../ui/button';
+import {
+  Text,
+  Box,
+  Button,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  BUTTON_VARIANT,
+  BUTTON_SIZES,
+} from '../../../component-library';
 
-export default function SnapRemoveWarning({ onCancel, onSubmit, snapName }) {
-  const t = useI18nContext();
-
-  const SnapRemoveWarningFooter = () => {
-    return (
-      <>
-        <Button
-          className="snap-remove-warning__footer-button"
-          type="default"
-          onClick={onCancel}
-        >
-          {t('nevermind')}
-        </Button>
-        <Button
-          id="popoverRemoveSnapButton"
-          className="snap-remove-warning__footer-button"
-          type="danger-primary"
-          onClick={onSubmit}
-        >
-          {t('removeSnap')}
-        </Button>
-      </>
-    );
-  };
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
 
+export default function SnapRemoveWarning({
```

Comment:
![image](https://github.com/MetaMask/metamask-extension/assets/1561200/536e8d43-4ab9-4a7c-a1bc-b7b70e7f1acf)
This component is no longer centered where it is used 🤔

Reply:
Shouldn't the body text font for this modal be our `bodyMd` baseline version? (@georgewrmarshall)

Reply:
The updated modal isn't centered it prevents it from jumping around when the content of the modal changes. Also yes the text should use `bodyMd`

Reply:
Updated text variant here https://github.com/MetaMask/metamask-extension/pull/19563/commits/20f698df12c88196235a936f332b4b0608af55f8